test/extended/networking/egressip_helpers.go

```diff
@@ -1273,23 +1281,22 @@ func sendProbesToHostPort(oc *exutil.CLI, proberPod *v1.Pod, url, targetProtocol
 	request := fmt.Sprintf("http://%s/dial?protocol=%s&host=%s&port=%d&request=%s", url, targetProtocol, targetHost, targetPort, randomIDStr)
 	var wg sync.WaitGroup
 	errChan := make(chan error, iterations)
+	defer close(errChan) // Ensure the channel is closed after all goroutines finish
```

Comment:
heads up, channels should automatically close when garbage collected on descope.